pheget/api/__init__.py

```diff
@@ -129,3 +129,24 @@ def variant_query(chrom: str, pos: int):
 
     results = {"data": data}
     return jsonify(results)
+
+
+@api_blueprint.route(
+    "/region/<string:chrom>/<int:start>-<int:end>/data/", methods=["GET"]
+)
```

Comment:
Can you help me understand the difference in requirements between this endpoint and `region_query` above, in the same file? I may be missing an important subtlety, and there are no unit tests to guide on the differences.

It seems they are overall very similar, except with minor differences in optional query params (tissue vs piponly), API response formatting (`{}` vs `{'data': {}`), and query options (`chr` prefix handling).

Overall it sort of leaves the impression that both endpoints *could* be consolidated into one. From a maintainability/API standpoint, it can be very jarring if every view in your app obeys different query syntax and formatting rules.

Reply:
You're correct, this was just a quick hack to test ways to populate the table. I thought one way to consolidate this is to add an option in the original region query API so that we can request data only vs. returning {"data": data}, but using the Tabulator ajaxResponse callback looks to be cleaner.